public/scripts/project/metadata.js

```diff
@@ -54,6 +54,20 @@ define(function(require) {
     });
   }
 
+  // Look up the title for this anonymous project
+  function getAnonymousTitle(root, callback) {
```

Comment:
Let's just have `getTitle()` and it knows what to do whether you're logged in or not.